server/README.md

```diff
@@ -0,0 +1,13 @@
+# Python server
+Usage:
+
+`python server.py [-p PORT] [-a ADDRESS| address]`
+
+address must be in form `your.address.com`, e.g. without `http` prefix.
```

Comment:
A nit pick: if it is called a hostname - there will be no need to explain the lack of protocol name in it.